puts/harness/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+cmake_minimum_required(VERSION 3.15)
+
+project(puts VERSION 0.1 LANGUAGES C)
+
+enable_testing()
+
+set(PUTS "")
+set(PUT_LIBS "")
+
+string(REPLACE "," ";" VENDORS "${VENDORS}")
```

Comment:
What is a vendor?

Reply:
That's a very good question. My current naming scheme is:
- the vendor library (or simply vendor) is the library we are testing (e.g. OpenSSL / OpenSSL 1.1.1k). This is why the `mk_vendor` script is called that.
- the harness is the wrapper code that allow us to orchestrate the fuzzing/trace execution on the vendor library (creating server/client, setting agents runtime configuration, ...)
- the PUT is the pair harness + vendor

I'm open to change in that regard. I think here is not the best place to have this discussion, but I would gladly discuss the matter in a specific issue. Or we can setup a meeting. In any case, I would like that we all agree on the naming before I start writing the documentation for tlspuffin, so that I can also rename the Rust traits/structs as necessary and keep the tiny bit of sanity I still maintain 🥺

Reply:
I thought about this and I agree with you. In this context it makes more sense to talk about a "library" that we want to wrap into our harness code. I modified it here and in the calling context of this CMakeLists.txt.